dashboard/app/assets/javascripts/client_state.js

```diff
@@ -37,10 +37,27 @@ var COOKIE_OPTIONS = {expires: dashboard.clientState.EXPIRY_DAYS, path: '/'};
 dashboard.clientState.reset = function() {
   $.removeCookie('progress', {path: '/'});
   $.removeCookie('lines', {path: '/'});
+  $.removeCookie('puzzle_rating', {path: '/'});
```

Comment:
Hmm, I see that you need the cookies because they have to get sent up to the server and you want to do that implicitly.  I guess since this lives a short time and isn't large it should be fine as is.